pom.xml

```diff
@@ -171,6 +171,7 @@
               <exclude>src/site/resources/download_lang.cgi</exclude>
               <exclude>src/test/resources/stringEscapeUtilsTestData.txt</exclude>
               <exclude>src/site/resources/release-notes/RELEASE-NOTES-*.txt</exclude>
+              <exclude>.travis.sh</exclude>
```

Comment:
It looks like, with a quick read of their License.md, that we would minimally need to include their license at the end of ours, but I would have to defer to others for that.